### DISCUSSION TITLE
Compare date/time and THEN XML files

### DISCUSSION THREAD

**Discussion by gerardsweeney** [WinMerge/winmerge#765] [Q&A]
Hi, folks..

I'm probably missing something REALLY obvious.

I'm comparing two deployment installers created by AutoDesk.
One is set to have the licence server set to server1, one is set to have the licence server set to server2.

If I do a compare by date/time, it shows a raft of XML files which it says "Files are different".
If I click "Compare" on each, a new window opens and says that the 2 files are identical.

Can I do a "batch compare within a compare"?

So.. Compare by date/time so that of the 33,000 files in the folder, it shows the 247 XML files it says are different.
And then compare each of those 247 XML files to find the ones which are actually different?

I tried changing the initial compare method to quick content, but it was taking forever to compare the 33,000 files.

Like I said - I've probably missed something REALLY obvious :)

Cheers,
Gerard

**Comment by sdottaka** [comment]
It cannot be done automatically, but you can change the "comparison method" after the comparison, so please follow the steps below to re-compare only the selected items.

1. Compare folders by Modified Date and Size

2. Change the compare method to Full Contents

![image](https://user-images.githubusercontent.com/98126/116764153-0b1b0c80-aa5b-11eb-8afe-2a50d0c8f8b0.png)

3. Select the XML files and re-compare by clicking Refresh Selected menu item

![image](https://user-images.githubusercontent.com/98126/116764168-238b2700-aa5b-11eb-8124-81c10b374390.png)

**Reply by gerardsweeney** [reply]
That worked an absolute treat - THANK YOU!
